pkg/tests/apis/secret/testdata/secure-value-generate.yaml

```diff
@@ -12,6 +12,5 @@ spec:
   keeper: my-keeper-1
   value: super duper secure
   decrypters:
-    - k6.grafana.app/app-name-1
-    - k6.grafana.app/app-name-2
-    - entity.grafana.app/*
+    - secrets/k6:decrypt
+    - secrets/synthetic-monitoring:decrypt
```

Comment:
Tests running with Enterprise would fail otherwise.

Reply:
What is `secrets/k6:decrypt` and where does it come from?